packages/typescript/internal/ts_project.bzl

```diff
@@ -246,7 +246,7 @@ def _ts_project_impl(ctx):
     # Don't provide DeclarationInfo if there are no typings to provide.
     # Improves error messaging if a ts_project needs declaration = True
     if len(typings_outputs) or len(ctx.attr.deps):
-        providers.append(declaration_info(depset(typings_outputs), ctx.attr.deps))
+        providers.append(declaration_info(depset(typings_outputs), depset([]), ctx.attr.deps))
```

Comment:
I wonder if the function signature could be changed so `depset([])` is the default for this new param and existing callers don't have to change